widget/embedded/src/components/Layout/Layout.tsx

```diff
@@ -52,7 +45,7 @@ function LayoutComponent(props: PropsWithChildren<PropTypes>, ref: Ref) {
   return (
     <Container
       ref={ref}
-      fixedHeight={fixedHeight}
+      auto={auto}
```

Comment:
I think `auto` doesn't make sense. Can you change it to a better name? 
I don't have a better alternative name but for example `autoAdjust`, `isContentBased`, `flexibleHeight` or something.